lib/MetaCPAN/Web/Test/HTML5/TreeBuilder.pm

```diff
@@ -4,7 +4,6 @@ use warnings;
 
 use parent 'HTML::TreeBuilder';
 
-use MetaCPAN::Web::Test::HTML5::Element::SVG;
```

Comment:
This seems wrong. This module is needed.

Reply:
It wasn't clear to me how it's needed in this file. I was only seeing the class name used in an `isa` check.